### ISSUE TITLE
Insufficient colour contrast when header logo focused

### ISSUE THREAD

**Issue by chrisrhymes** [FutureNorthants/northants-design-system#544]
The header logo has a gold outline when it is focused. This provides insufficient contrast with the white background in the new header layout to meet the AA standard. 

https://index.silktide.com/website/west-northamptonshire/january-2025/recommendations/ensure-focus-appearance

Add the default link focus state where it has the gold and black outline to improve the contrast against the white header background, then retest using the silktide browser plugin.
